common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -209,7 +219,7 @@ object GettyCreditParser extends ImageProcessor with GettyProcessor {
   def knownGettyCredits(image: Image, credit: String): Image =
     gettyCredits.find(_.toLowerCase == credit.toLowerCase) match {
       case collection @ Some(_) => image.copy(
-        usageRights = gettyAgencyWithCollection(collection)
+        usageRights = gettyAgencyWithCollection(collection).getOrElse(NoRights)
```

Comment:
So this is a weird one - basically it means that if we haven't defined a canonical name for the "getty" supplier we'll default to `NoRights` - this is because we now need a single source of truth for supplier names as they are used in multiple places - so this basically means if you write some code that doesn't have a known supplier it'll end up with `NoRights` but obviously not here because we have!